build.gradle

```diff
@@ -19,17 +19,24 @@ dependencies {
     compile('org.springframework.boot:spring-boot-starter-validation')
     compile('org.springframework.boot:spring-boot-starter-thymeleaf')
     compile('org.postgresql:postgresql')
-    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
-    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'
-    compile 'org.springframework.plugin:spring-plugin-core:1.2.0.RELEASE'
-    compile group: 'com.google.guava', name: 'guava', version: '27.0-jre'
+    compile('io.springfox:springfox-swagger2:2.9.2')
+    compile('io.springfox:springfox-swagger-ui:2.9.2')
+    compile('org.springframework.plugin:spring-plugin-core:1.2.0.RELEASE')
+    compile('com.google.guava:guava:27.0-jre')
+    compileOnly('org.projectlombok:lombok:1.18.16')
+    annotationProcessor('org.projectlombok:lombok:1.18.16')
 
+    testCompileOnly('org.projectlombok:lombok:1.18.16')
+    testAnnotationProcessor('org.projectlombok:lombok:1.18.16')
     testCompile('junit:junit:4.12')
     testCompile('com.github.tomakehurst:wiremock:2.27.2')
     testImplementation('com.h2database:h2:1.4.200')
     testImplementation('org.springframework.security:spring-security-test')
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
-        exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+        exclude group: 'org.junit.vintage', module: 'jun' +
```

Comment:
Leave it like before. I don't like this haha

Reply:
And here what happened, I don't understand how it happened

Reply:
corrected